bindings/wasm/manifold.d.ts

```diff
@@ -20,10 +20,9 @@ interface SealedFloat32Array<N extends number> extends Float32Array {
   length: N;
 }
 
-type Vec2 = [number, number];
-type Vec3 = [number, number, number];
-type Vec4 = [number, number, number, number];
-type Matrix3x4 = [Vec3, Vec3, Vec3, Vec3];
+type Vec2 = number[2];
+type Vec3 = number[3];
+type Mat4 = number[16];
```

Comment:
Oops, pretty sure this is wrong.

Reply:
Maybe we should do this: https://stackoverflow.com/a/59906630

It is interesting that typescript did not put this in their type definitions even though this should be a rather common usecase...

Reply:
And maybe we should explicitly say that the matrix is column-major? Otherwise users may be confused about this.

Reply:
I went back mostly to what we had before with tuple types. They seem to work the best and give pretty good type safety. The only drawback is I had to use `vec2.fromValues(1, 1)` instead of `[1, 2]` whenever I wanted to save the variable, but that doesn't seem too bad - and that's just because I can't cast it in our JS tests like users can in the editor.